code-challenges/filter/challenges-08.test.js

```diff
@@ -0,0 +1,271 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named oddValues that, given an array of integers as input, uses filter to return an array containing only the odd integers.
+
+For example, oddValues([1,2,3]) returns [1,3].
+------------------------------------------------------------------------------------------------ */
+
+const oddValues = (arr) => {
+  // Solution code here...
+  return arr.filter(index => index % 2 !== 0);
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named filterStringsWithVowels that, given an array of strings as input, uses filter to return an array with only words that contain vowels.
+
+The callback function to filter should include or utilize a regular expression pattern.
+
+For example, filterStringsWithVowels('gregor','hound','xyz') returns ['gregor', 'hound'].
+------------------------------------------------------------------------------------------------ */
+
+
+const filterStringsWithVowels = (arr) => {
+  
+  return arr.filter(index => index.match(/[aeiou]/g));
```

Comment:
You don't need to `g` flag in the regex, you only need to see if the string has one vowel. It would have been better to put the `i` flag instead so that this function would work on uppercase vowels as well